ci/properties/azure-data-factory.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Azure Data Factory",
+    "creator": "Microsoft Azure",
+    "description": "Build and validate Azure Data Factory resources",
+    "iconName": "azure-data-factory",
+    "categories": ["Continuous integration", "Azure", "Data Factory"]
```

Comment:
Are there any criteria to detect if a repository has Azure Data Factory resources? `categories` field is used to examine how relevant a template is to any concerned repository.

Reply:
@NinadKavimandan one criteria could be detecting if a repository has a `package.json` file that represents the Data Factory utilities package:

```json
{
    "scripts":{
        "build":"node node_modules/@microsoft/azure-data-factory-utilities/lib/index"
    },
    "dependencies":{
        "@microsoft/azure-data-factory-utilities":"^0.1.5"
    }
} 
```

Reply:
Is there a built-in way to check if certain files exists in the repository? Any suggestion?

Reply:
One potential approach I've seen is contributing to [languages.yml](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) on linguist to look for a `package.json` file that contains the `@microsoft/azure-data-factory-utilities` dependency. Not sure if there's a way to make this validation on linguist.

Reply:
I've updated the `categories` section to look for `.json` files.

Reply:
One potential improvement in the future is contributing to [languages.yml](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) on linguist or using another way to look for a `.json` file that contains the `@microsoft/azure-data-factory-utilities` keyword.